R/clean.R

```diff
@@ -50,7 +50,7 @@ find_meta_stopwords <- function(metadata,
                                 stop_cols =  list(
                                   'author1' = c('first_name', 'last_name'),
                                   'author2' = c('first_name_2', 'last_name_2'),
-                                  'title'  = 'title'
+                                  'title'  = c('full_title', 'title', 'subtitle')
```

Comment:
this function treats each element of a list as an aggregate of words  - we want it for authors because names of authors need to be removed only when they're first and last name are written together. 

So consequently, the line `c('full_title', 'title', 'subtitle')` would search for full title followed by title followed by subtitle. Probably not what we want . I think we need either: 

```suggestion
                                  'title'  = 'title',
                                  'subtitle' = 'subtitle',
                                  'full_title' = 'full_title  # or 'full_title' = c('title','subtitle')
                                  
                                  
```